src/performance/contentful.ts

```diff
@@ -0,0 +1,20 @@
+import { Event } from "@clarity-types/data";
+import { LargestContentfulData, LargestContentfulPaint } from "@clarity-types/performance";
+import { schedule } from "@src/core/task";
+import encode from "./encode";
+
+export let data: LargestContentfulData = null;
```

Comment:
Add link to [https://wicg.github.io/largest-contentful-paint/](https://wicg.github.io/largest-contentful-paint/)
interface LargestContentfulPaint : PerformanceEntry {
    readonly attribute DOMHighResTimeStamp renderTime;
    readonly attribute DOMHighResTimeStamp loadTime;
    readonly attribute unsigned long size;
    readonly attribute DOMString id;
    readonly attribute DOMString url;
    readonly attribute Element? element;
    [Default] object toJSON();
};

Reply:
Same as above. Already in performance.d.ts where interface are defined.

Reply:
Yeah, but that's not a spec, just some TypeScript. For me having the spec link is nice as that's where I go to see what the data should mean and what is "correct". 

---
In reply to: [347636727](https://github.com/microsoft/clarity-js/pull/160#discussion_r347636727) [](ancestors = 347636727)